modules/programs/git.nix

```diff
@@ -476,8 +486,11 @@ in {
       home.packages = [ pkgs.difftastic ];
 
       programs.git.iniContent = let
-        difftCommand =
-          "${pkgs.difftastic}/bin/difft --color ${cfg.difftastic.color} --background ${cfg.difftastic.background}";
+        difftCommand = "${pkgs.difftastic}/bin/difft" + concatStringsSep " " [
```

Comment:
This will produce `…/bin/difft--color …` which will cause an error when git tries to run it.

Reply:
thank you!, see https://github.com/nix-community/home-manager/pull/3089